src/hccomands.py

```diff
@@ -28,13 +28,7 @@ async def fufu(ctx):
 
 @commands.command()
 async def buildjhin(ctx):
-    await ctx.send("Jhin 10.10 Core Build:")
-    await ctx.send("Stormrazor")
-    await ctx.send("Infinity Edge")
-    await ctx.send("Rapid Firecannon")
-    await ctx.send("----------------------------------------")
-    await ctx.send("Boots: Swiftness")
-    await ctx.send("Keystone: Fleet Footwork")
-
-
+    Jhinbuild = "Jhin 10.10 Core Build: \n Stormrazor \n Infinity Edge \n Rapid Firecannon \n ---------------------------------------- \n Boots: Swiftness \n Keystone: Fleet Footwork"
```

Comment:
```suggestion
    jhinbuild = 'Jhin 10.10 Core Build: \n Stormrazor \n Infinity Edge \n Rapid Firecannon \n ---------------------------------------- \n Boots: Swiftness \n Keystone: Fleet Footwork'
```
Variablen werden normalerweise klein geschrieben in python. Man benutzt selten die doppelten Anführungszeichen.